frontend/components/home/Banner.tsx

```
@@ -0,0 +1,157 @@
+                                            viewBox='0 0 61 68'
+                                            fill='none'
+                                            xmlns='http://www.w3.org/2000/svg'
+                                            aria-label='learning-curve'
+                                        >
+                                            <path
+                                                d='M10.9456 42.4604C12.35 35.8453 15.0985 20.3798 14.8574 11.4385'
+                                                stroke='#031333'
+                                                strokeWidth='3.07158'
+                                                strokeLinejoin='round'
+                                                style={{ strokeDasharray: '32, 34', strokeDashoffset: 0 }}
+                                            ></path>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)